.travis.yml

```diff
@@ -9,7 +9,7 @@ env:
   - PACKAGE="charrua-core"
   - TESTS=true
   - PINS="charrua-client:. charrua-unix:."
-  - DEPOPTS="charrua-client charrua-unix"
+  - DEPOPTS="charrua-client charrua-client-lwt charrua-client-mirage charrua-unix"
```

Comment:
I thought the build was not using depopts anymore (since the switch to jbuilder)?